docs/graphx-programming-guide.md

```diff
@@ -420,24 +421,24 @@ val defaultUser = ("John Doe", "Missing")
 val graph = Graph(users, relationships, defaultUser)
 // Notice that there is a user 0 (for which we have no information) connected to users
 // 4 (peter) and 5 (franklin).
-graph.triplets.map(
-    triplet => triplet.srcAttr._1 + " is the " + triplet.attr + " of " + triplet.dstAttr._1
-  ).collect.foreach(println(_))
+graph.triplets.map {
```

Comment:
I don't agree with these changes. A scope is not required here. It was correct as-is.